src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -39,6 +40,12 @@ public abstract partial class Frame : FrameContext, IFrameControl
         private static readonly byte[] _bytesDate = Encoding.ASCII.GetBytes("Date: ");
         private static readonly byte[] _bytesEndHeaders = Encoding.ASCII.GetBytes("\r\n\r\n");
 
+        private static readonly Vector<byte> _vectorCRs = new Vector<byte>((byte)'\r');
```

Comment:
Though there is a lot of weird code for the constructor in the jitter so I'm not actaully sure what it does https://github.com/dotnet/coreclr/blob/master/src/jit/simd.cpp#L1874 but whatever it is - its avoiding that :D